coil-core/src/androidMain/kotlin/coil3/RealImageLoader.kt

```diff
@@ -80,6 +83,16 @@ internal actual fun ComponentRegistry.Builder.addAndroidComponents(
         .add(DrawableFetcher.Factory())
         .add(BitmapFetcher.Factory())
         // Decoders
+        .apply {
+            val canUseImageDecoder = options.bitmapFactoryExifOrientationPolicy == ExifOrientationPolicy.RESPECT_PERFORMANCE
+            if (canUseImageDecoder && SDK_INT >= 28) {
+                add(
+                    StaticImageDecoderDecoder.Factory(
```

Comment:
Let's always use `StaticImageDecoderDecoder.Factory` on API 29 and up even if it ignores `ExifOrientationPolicy.RESPECT_PERFORMANCE`. I think it might make sense to either remove or deprecate this API in Coil 3 as it has low usage and can't be supported by `ImageDecoder`.

Reply:
Done